mixin.mk

```diff
@@ -10,8 +10,7 @@ PERMALINK ?= $(shell git describe --tags --exact-match --match v* &> /dev/null &
 
 LDFLAGS = -w -X $(PKG)/pkg.Version=$(VERSION) -X $(PKG)/pkg.Commit=$(COMMIT)
 GO = GO111MODULE=on go
-# I am using both ways to disable http/2 which is causing us massive instability.  Hopefully one of them sticks.
-XBUILD = CGO_ENABLED=0 GO111MODULE=on GODEBUG=http2client=0 $(GO) build -ldflags '$(LDFLAGS)' -tags nethttpomithttp2
+XBUILD = CGO_ENABLED=0 GO111MODULE=on $(GO) build -ldflags '$(LDFLAGS)'
```

Comment:
I just realized that now that v1 is targeting 1.16 we can remove all these GO111MODULE=on stuff in our projects. I'll do that today.